src/components/organisms/AddForm/AddForm.jsx

```diff
@@ -0,0 +1,71 @@
+import { Formik, Form, Field } from 'formik';
```

Comment:
Generally I like the way you did it, I know that's not the high scalable form component which can be generic as fuck, but for our purposes it's looking really good, and I see that the code looks like it can be expanded really easily so if in future we will be willing to add other forms, there's shouldn't be a problem with expanding that component, good job